src/Umbraco.Community.AzureSSO/MicrosoftAccountBackOfficeExternalLoginProviderOptions.cs

```diff
@@ -126,5 +128,10 @@ private string DisplayName(ClaimsPrincipal claimsPrincipal, string defaultValue)
 
 			return !string.IsNullOrWhiteSpace(displayName) ? displayName : defaultValue;
 		}
+
+		public void Configure(BackOfficeExternalLoginProviderOptions options)
+		{
+			// TODO : What do we do here
```

Comment:
I think we should be maintaining the previous behaviour here, or if that isn't possible then let's get rid of this redundant method.

Reply:
Was just thinking on this. It's enforced by the interface, but doesn't appear to be used in this config